src/components/Wallet/WalletGenerator.jsx

```diff
@@ -51,9 +51,7 @@ import {
   SET_CLIENT_PASSWORD,
   SET_CLIENT_PASSWORD_ERROR,
 } from "../../actions/clientActions";
-
-const MAX_TRAILING_EMPTY_NODES = 20;
```

Comment:
These same values were used in another file with a note to use a shared constant somewhere. We've since created a "constants" file for use in components, so I moved these there.